--- aster-full-src-11.5.0/SRC/med-3.0.7/config/med_check_hdf5.m4.orig	2013-10-22 15:53:08.000000000 +0200
+++ aster-full-src-11.5.0/SRC/med-3.0.7/config/med_check_hdf5.m4	2014-02-28 04:58:15.078863474 +0100
@@ -26,6 +26,12 @@ AC_DEFUN([_MED_DEFINE_HDF5_ARGS],[
 ])
 
 AC_DEFUN([_MED_BEFORE_ENABLE_PMED], [
+if uname -m | grep 64; then
+  LIB_LOCATION_SUFFIX="64"
+else
+  LIB_LOCATION_SUFFIX=""
+fi
+
 AC_REQUIRE([_MED_DEFINE_HDF5_ARGS])
 
 dnl ## If user hasn't provided hdf5 path try using "$HDF5HOME" 
@@ -39,8 +45,8 @@ dnl #If user specified a specific hdf5ho
          AC_MSG_ERROR([The directory |$hdf5home| doesn't exist, please verify either HDF5HOME or --with-hdf5=<path>])
        fi
 
-       if test ! -d "$hdf5home/lib" ; then
-         AC_MSG_ERROR([The directory |$hdf5home/lib| doesn't exist, please verify either HDF5HOME or --with-hdf5=<path>])
+       if test ! -d "$hdf5home/lib${LIB_LOCATION_SUFFIX}" ; then
+         AC_MSG_ERROR([The directory |$hdf5home/lib${LIB_LOCATION_SUFFIX}| doesn't exist, please verify either HDF5HOME or --with-hdf5=<path>])
        fi
 
        if test ! -d "$hdf5home/include" ; then
@@ -65,7 +71,7 @@ AC_REQUIRE([MED_ENABLE_PMED])
       LIBS_save="$LIBS"
       CPPFLAGS_save="$CPPFLAGS"
 
-      LDFLAGS="-L$hdf5home/lib $LDFLAGS"
+      LDFLAGS="-L$hdf5home/lib{LIB_LOCATION_SUFFIX} $LDFLAGS"
       CPPFLAGS="-I$hdf5home/include $CPPFLAGS "
 
       AX_ABSOLUTE_HEADER([H5public.h])
@@ -105,7 +111,7 @@ dnl          AC_MSG_WARN([BE CAREFUL, we
 dnl        fi
 
       HDF5_CPPFLAGS="$HDF5_CPPFLAGS -I$hdf5home/include"
-      HDF5_LDFLAGS="-L$hdf5home/lib"
+      HDF5_LDFLAGS="-L$hdf5home/lib{LIB_LOCATION_SUFFIX}"
       LDFLAGS="$LDFLAGS_save"
       CPPFLAGS="$CPPFLAGS_save"
       LIBS="$LIBS_save"
