--- aster-full-src-11.5.0/SRC/med-3.0.7/python/Makefile.am~	2014-02-28 05:19:28.798888338 +0100
+++ aster-full-src-11.5.0/SRC/med-3.0.7/python/Makefile.am	2014-02-28 05:36:59.888908856 +0100
@@ -83,7 +83,7 @@ pkgpyexec_LTLIBRARIES     += _medlibrary
 
 _medlibrary_la_LDFLAGS         = -module -no-undefined
 #_medlibrary_la_LDFLAGS = -avoid-version -module
-_medlibrary_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medlibrary_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medlibrary_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medlibrary_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medlibrary_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -104,7 +104,7 @@ pkgpyexec_PYTHON   +=  medfile.py
 pkgpyexec_LTLIBRARIES     += _medfile.la
 
 _medfile_la_LDFLAGS         = -module -no-undefined
-_medfile_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medfile_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medfile_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medfile_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medfile_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -126,7 +126,7 @@ pkgpyexec_LTLIBRARIES     += _medenum.la
 
 _medenum_la_LDFLAGS         = -module -no-undefined
 
-_medenum_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medenum_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medenum_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medenum_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medenum_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -149,7 +149,7 @@ pkgpyexec_LTLIBRARIES     += _medmesh.la
 
 _medmesh_la_LDFLAGS         = -module -no-undefined
 
-_medmesh_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medmesh_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medmesh_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medmesh_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medmesh_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -172,7 +172,7 @@ pkgpyexec_LTLIBRARIES     += _medfield.l
 
 _medfield_la_LDFLAGS         = -module -no-undefined
 
-_medfield_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medfield_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medfield_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medfield_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medfield_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -194,7 +194,7 @@ pkgpyexec_LTLIBRARIES     += _medfamily.
 
 _medfamily_la_LDFLAGS         = -module -no-undefined
 
-_medfamily_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medfamily_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medfamily_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medfamily_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medfamily_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -216,7 +216,7 @@ pkgpyexec_LTLIBRARIES     += _medequival
 
 _medequivalence_la_LDFLAGS         = -module -no-undefined
 
-_medequivalence_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medequivalence_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medequivalence_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medequivalence_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medequivalence_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -239,7 +239,7 @@ pkgpyexec_LTLIBRARIES     += _medenumtes
 
 _medenumtest_la_LDFLAGS         = -module -no-undefined
 
-_medenumtest_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medenumtest_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medenumtest_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medenumtest_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medenumtest_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -261,7 +261,7 @@ pkgpyexec_LTLIBRARIES     += _medprofile
 
 _medprofile_la_LDFLAGS         = -module -no-undefined
 
-_medprofile_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medprofile_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medprofile_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medprofile_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medprofile_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -283,7 +283,7 @@ pkgpyexec_LTLIBRARIES     += _medlocaliz
 
 _medlocalization_la_LDFLAGS         = -module -no-undefined
 
-_medlocalization_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medlocalization_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medlocalization_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medlocalization_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medlocalization_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -305,7 +305,7 @@ pkgpyexec_LTLIBRARIES     += _medlink.la
 
 _medlink_la_LDFLAGS         = -module -no-undefined
 
-_medlink_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medlink_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medlink_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medlink_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medlink_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -327,7 +327,7 @@ pkgpyexec_LTLIBRARIES     += _medparamet
 
 _medparameter_la_LDFLAGS         = -module -no-undefined
 
-_medparameter_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medparameter_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medparameter_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medparameter_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medparameter_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -349,7 +349,7 @@ pkgpyexec_LTLIBRARIES     += _medfilter.
 
 _medfilter_la_LDFLAGS         = -module -no-undefined
 
-_medfilter_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medfilter_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medfilter_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medfilter_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medfilter_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -371,7 +371,7 @@ pkgpyexec_LTLIBRARIES     += _medsubdoma
 
 _medsubdomain_la_LDFLAGS         = -module -no-undefined
 
-_medsubdomain_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medsubdomain_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medsubdomain_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medsubdomain_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medsubdomain_la_CPPFLAGS       += $(AM_CPPFLAGS)
@@ -393,7 +393,7 @@ pkgpyexec_LTLIBRARIES     += _medinterp.
 
 _medinterp_la_LDFLAGS         = -module -no-undefined
 
-_medinterp_la_LIBADD          = $(top_builddir)/src/libmed.la
+_medinterp_la_LIBADD          = $(top_builddir)/src/libmed.la -lpython2.7
 _medinterp_la_CPPFLAGS        = $(SWIG_PYTHON_CPPFLAGS)
 _medinterp_la_CPPFLAGS       += $(PYTHON_CPPFLAGS)
 _medinterp_la_CPPFLAGS       += $(AM_CPPFLAGS)
