--- aster-full-src-11.5.0/SRC/scotch-5.1.11_esmumps/src/libscotch/parser.c.strfmt~	2014-02-28 04:53:05.198857425 +0100
+++ aster-full-src-11.5.0/SRC/scotch-5.1.11_esmumps/src/libscotch/parser.c	2014-02-28 04:53:11.148857541 +0100
@@ -590,15 +590,15 @@ FILE * const                stream)
     case STRATTESTMUL :                           /* Multiplication operator */
     case STRATTESTMOD :                           /* Modulus operator        */
       i = (test->data.test[0]->typetest < test->typetest) ? 1 : 0;
-      fprintf (stream, strattestsavepa[i][0]);
+      fprintf (stream, "%s", strattestsavepa[i][0]);
       o = stratTestSave (test->data.test[0], stream);
-      fprintf (stream, strattestsavepa[i][1]);
+      fprintf (stream, "%s", strattestsavepa[i][1]);
       if (o == 0) {
         fprintf (stream, "%c", strattestsaveop[test->typetest]);
         i = (test->data.test[1]->typetest < test->typetest) ? 1 : 0;
-        fprintf (stream, strattestsavepa[i][0]);
+        fprintf (stream, "%s", strattestsavepa[i][0]);
         stratTestSave (test->data.test[1], stream);
-        fprintf (stream, strattestsavepa[i][1]);
+        fprintf (stream, "%s", strattestsavepa[i][1]);
       }
       break;
     case STRATTESTVAL :                           /* Constant value */
